pages/wg/outreach.md

```diff
@@ -20,77 +20,110 @@ bi-weekly virtual meetings. To get involved, visit the
 slack or contact the
 <a href="mailto:wg-outreach@us-rse.org">Outreach working group list</a>.
 
-# Work Products
+**Co-chairs**: Miranda Mundt, Joshua Teves
 
-The US-RSE Outreach working group is new as of 2022. The following events were
-either completed before the creation of the official working group or are
-initiatives for future work.
+# Work Products
 
-## Existing Efforts
+The official US-RSE Outreach working group began in 2022. The group has produced
+and hosts the following media, documents, and events.
 
-### RSE Stories
+## RSE Stories
 
 Created by Vanessa Sochat, the [RSE Stories podcast](https://us-rse.org/rse-stories/)
 aims to cast a spotlight on the differing backgrounds and paths professionals
 have taken to become RSEs and help shape our understanding of what an RSE really is.
 
 The podcast has existed since September 2019 and is looking for more volunteers
-both for interviewees and hosts.
+for interviewees, hosts, and audio editors.
 
-### Conference and Workshop Participation
+## Conference/Workshop Participation and Organization
 
 US-RSE has had a presence at several conferences and workshops throughout
-its lifetime. For example:
+its lifetime and continues to find new venues. A non-exhaustive list of
+previous or upcoming events:
 
+- Supercomputing 2022: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2022/), Student Program Orientation
+- eScience 2022: [RSEs in eScience Workshop](https://us-rse.org/rse-escience-2022/)
+- US-RSE Virtual Workshop 2022: [Make it happen! Towards a diverse and sustainable future!](https://us-rse.org/virtual-workshop-2022/)
+- SciPy 2022: [(Re)Introducing the US-RSE Association to SciPy BoF](https://www.scipy2022.scipy.org/update-conference-schedule)
 - Supercomputing 2021: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2021/), Student Program Orientation
+- US-RSE Virtual Workshop 2021: [A Path Forward for Research Software Engineers](https://us-rse.org/virtual-workshop-2021/)
 - PEARC21: [RSE Career Panel](https://us-rse.org/2021-07-20-pearc21-panel/)
 - Supercomputing 2020: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2020/)
 - PEARC20: [Research Software Engineers Community Workshop](https://us-rse.org/events/2020/2020-07-pearc20/)
 - PEARC19: Building a Community of Research Software Engineers Birds of a Feather - [Proposal](https://ssl.linklings.net/conferences/pearc/pearc19_program/views/includes/files/bof111s1.pdf), [Event](https://us-rse.org/events/2019/pearc-2019/)
 - EDUCAUSE 2019: [Supporting Research Software and Research Software Engineers Poster](https://events.educause.edu/annual-conference/2019/agenda/supporting-research-software-development)
 
-### External Collaboration
 
-US-RSE has collaborated with other organizations in the past for events
-such as the [Early Career Panel](https://academicdatascience.org/cdn/early-career-panel)
-([Academic Data Science Alliance](https://academicdatascience.org/)).
+## Speaker Series
```

Comment:
I don't know how to fix this, but I again feel like this higher level heading looks lost and I got slightly confused during my first read. Maybe number the two speaker series 1 and 2? Maybe have a horizontal line before the speaker series heading so it's clearly a new section?